six/modules/c++/cphd/include/cphd/CPHDWriter.h

```diff
@@ -84,11 +84,15 @@ class CPHDWriter
      *         data in chunks. Otherwise you should use addImage and write.
      *         This will internally set the number of bytes per VBP.
      *
-     *  \pathname The desired pathname of the file.
-     *  \vbm The vector based metadata to write.
+     *  \param pathname The desired pathname of the file.
+     *  \param vbm The vector based metadata to write.
+     *  \param classification The classification of the file
```

Comment:
I’ve made the changes.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Friday, February 26, 2016 12:27 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>
Subject: Re: [six-library] Add optional classification and releaseInfo parameters to writing functions (#51)


In six/modules/c++/cphd/include/cphd/CPHDWriter.h<http://cp.mcafee.com/d/k-Kr418SyM-U_ssUCCrKrpop73zhO--rv7cIcECzAQsCQrLnusd7dPhOYrhhphjvd7bz3VIhH3pAZxOLX2walgRKDbCRkdrFOVJPZuKwU_R-vj7fIcKtuVtdd4tOdS1RS7PhPR4kRHFGTooVkffGhBrwqrhdFCXYyMCY-ehojd79KVIDeqR4IMzapdIxO-g8Wh-IEjOAwYKv8x7oBImrffNZ4349kITiAGM5oLI9zCnT1NK5o-Ium1h6kOrp3BPr1EVojdwLQzh0qmWi3OVYy4tymdbFEw4443h1lqsGMQYQgmfSB9eOJIVlwq8dwwq88-v-QYdwXkQgbr7lvDYq6y2tjh0ozasE4jh0WT2QVlxgQg6yw37Qjhird79EVdPHC1cKhg>:

> @@ -84,11 +84,15 @@ class CPHDWriter

>       *         data in chunks. Otherwise you should use addImage and write.

>       *         This will internally set the number of bytes per VBP.

>       *

> -     *  \pathname The desired pathname of the file.

> -     *  \vbm The vector based metadata to write.

> +     *  \param pathname The desired pathname of the file.

> +     *  \param vbm The vector based metadata to write.

> +     *  \param classification The classification of the file

Add a note to these comments and the other ones that these two fields are optional and by default the CPHD will not be populated with these values

—
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/FZsScyhJ5xZN-UVNdcTsSOMOe76zBZYS-epophd79EVdETuKYUqerCzBUSyyOyC-qen67Pozm6P9X3BvS50kGxHtendGEqTjBPrDWZt1N_HY-CevopsWZOWqq8XArI3HIfCzDG8FHnjlKMNOEuvkzaT0QSCrjdTV5xdVYsyMCqejtPpesRG9px6kOrp3BYwhQzZpgDB91Vs-h2eNboISuvzWeeOf-w6hvoj7cLK3zsaNZoYI2ycFASO7bCS3hOMCr1vF6y0QJQA7BPV48X4Iqnjh08886y2GQVlxFVEwIvJaitBrpOH0Qgr10QghY_ZFUr1SFEwmSeG_fUQd44WCy0N6kVg8Cy1RK5FOH2xEwd506fECyASqejhOrsFIb065I>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.